dev/com.ibm.ws.concurrent.mp.cdi/src/com/ibm/ws/concurrent/mp/cdi/context/package-info.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+@Version("1.0.0")
+@TraceOptions(traceGroup = "concurrent")
```

Comment:
Typically context types are owned by the componentry that the represent the context for (in this case CDI), not by the context propagation infrastructure (MP Concurrency).  Wouldn't this make more sense with a CDI trace string and package?  For example, if you trace concurrency, you don't get all of the security context, classloading context and z/OS context information.  You have to trace those specific components if you want that.

Reply:
Will change this to the CDI trace group, `JCDI`, in a follow-on PR

Reply:
I need to understand this a bit more. Will this context propagation be always enabled in CDI? Is this functionality only enabled when both concurrency and cdi feature specified?

Reply:
this code is in an auto-feature that is only enabled when `cdi-2.0` and `mpConcurrency-1.0` are enabled